sklearn/titanic/README.md

```diff
@@ -1,35 +1,43 @@
+* have a [*GCP*](https://cloud.google.com/) account, and [download](https://cloud.google.com/sdk/), install, and [configure](https://cloud.google.com/sdk/gcloud/reference/config/) the *Google Cloud SDK* on our local computer.
 
-* [create a project](https://cloud.google.com/sdk/gcloud/reference/projects/create) on GCP and set the peoject and zone properties as instructed [here](https://cloud.google.com/sdk/gcloud/reference/config/set).
+* [create a project](https://cloud.google.com/sdk/gcloud/reference/projects/create) on *GCP* and set the peoject and zone properties as instructed [here](https://cloud.google.com/sdk/gcloud/reference/config/set).
 
 * set these environment variables on the local computer:
 	* $MY_INSTANCE: the name of the new instance (e.g. titanic_trainer).
-	* $MY_BUCKET: the bucket name to be created on GCS (e.g. titanic_bucket).
+	* $MY_BUCKET: the bucket name to be created on *Google Cloud Storage (GCS)* (e.g. titanic_bucket).
 	* $MY_FOLDER: the folder name to be used in the bucket (e.g. titanic_folder).
 
 ## Steps:
-We will create a Google Compute Engine instance, configure it, and use it to train a model. While some of the steps can be done using the web portal [here](https://pantheon.corp.google.com), we will try to accomplish this using the SDK and mainly the [gcloud](https://cloud.google.com/sdk/gcloud/) command. Unless otherwise specified as a comment, all the commands are to be run locally.
+We will create a [VM instance](https://cloud.google.com/compute/docs/instances/), configure it, and use it to train a model. While some of the steps can be done using the web portal [here](https://pantheon.corp.google.com/compute/instances), we will try to accomplish this using the SDK and mainly the [gcloud](https://cloud.google.com/sdk/gcloud/) command. Unless otherwise specified as a comment, all the commands are to be run locally.
 
 ##### 1. Create a Google Compute Engine Instance
-First, we should create a GCE instance that we can use to train our model on. In this case, we can rely on many of the default arguments and simply create the instance:
+First, we should create a *GCE* instance that we can use to train our model on. In this case, we can rely on many of the default arguments and simply create the instance:
 
 ```bash
 gcloud compute instances create $MY_INSTANCE
 ```
-which creates an instance of type *n1-standard-1* (1 CPU, 3.75GB of RAM), and a *Debian* disk image. We can however specify a different machine type or image. For example, the following command creates an *Ubuntu* instance of type *n1-standard-2*:
+which creates an instance of type *n1-standard-1* (1 CPU, 3.75GB of RAM), a *Debian* disk image, and with no scopes. This is fine as long as we need not the instance access *GCS*. Otherwise, we will have to provide the right scope:
```

Comment:
Not sure what you mean for this line: "This is fine as long as we need not the instance access *GCS*."